client/src/index.css

```diff
@@ -323,6 +323,108 @@ footer .icon i {
   margin: 20px auto;
 }
 
+.controls {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 100%;
+  margin-bottom: 1rem;
+}
+
+.controls label {
+  display: flex;
+  align-items: center;
+}
+
+.controls select,
+.controls input {
+  font-size: 1rem;
+  padding: 8px;
+  margin: 0 8px;
+  border: 1px solid #ddd;
+  border-radius: 4px;
+}
+
+.sort-order {
+  margin-left: 8px;
+  cursor: pointer;
+  transition: color 0.3s ease;
+}
+
+.controls button {
+  font-size: 1rem;
+  background: var(--primary);
+  color: #fff;
+  border: none;
+  padding: 8px 12px;
+  border-radius: 4px;
+  cursor: pointer;
+  transition: background 0.3s ease;
+}
+
+.controls button:hover {
```

Comment:
use different shade of primary btns to keep styles consistent. It must be their in some variable, add new var if needed but keep it blue only

Reply:
Done